pkgs/development/tools/doctl/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "doctl-${version}";
+  version = "1.3.1";
+  rev = "a57555c195d06bc7aa5037af77fde0665ad1231f";
+  goPackagePath = "github.com/digitalocean/doctl";
+
+  src = fetchFromGitHub {
+    owner = "digitalocean";
+    repo = "doctl";
+    rev = "${rev}";
+    sha256 = "03z652fw0a628gv666w8vpi05a4sdilvs1j5scjhcbi82zsbkvma";
+  };
+
+  meta = {
+    description = "A command line tool for DigitalOcean services";
+    homepage = "https://github.com/digitalocean/doctl";
+    license = stdenv.lib.licenses.apache2;
```

Comment:
The `meta` attribute is not taken into account when computing the output hash, so changes to `meta` won't trigger a rebuild. You can do
`nix-instantiate --eval --strict -A foo.meta` to force evaluation of the `meta` attribute.